interface-definitions/service-ids-ddos-protection.xml.in

```diff
@@ -70,17 +70,34 @@
                   <multi/>
                 </properties>
               </leafNode>
-              <node name="mode">
+              <leafNode name="mode">
                 <properties>
                   <help>Traffic capture modes</help>
```

Comment:
We usually use singulars in command descriptions< "Traffic capture _mode_".

Reply:
Fixed